.semaphore/semaphore.yml

```diff
@@ -0,0 +1,32 @@
+version: v1.0
+name: "Terraform-Github-Organization CI Pipeline"
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+blocks:
+
+  - name: "Build"
+    task:
+      prologue:
+        commands:
+          - export BUILD_VERSION=${SEMAPHORE_GIT_SHA:0:7}
+          - checkout
+      jobs:
+        - name: "Build Docker Image"
+          commands:
+            - make docker-build
+            - make docker-save
+            - cache store docker-image-$BUILD_VERSION terraform-github-organization-$BUILD_VERSION.tar
```

Comment:
is there a way to get this from a public docker registry?

Reply:
I haven't researched it yet but I guess so. That would also be one of the next logical steps to me.